style.css

```diff
@@ -102,6 +103,6 @@ section {
   transition: 0.4s;
 }
 
-ul {
-  display: flex;
+li {
+  display: inline;
```

Comment:
- [ ] [OPTIONAL] Great job aligning the icons as the previous reviewer suggested 👍 . Kindly make sure the icons are spaced properly and start on the same line as the text 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/191866363-8a9df6bc-83c4-4cfa-911c-84db351e36e9.png)  |  ![image](https://user-images.githubusercontent.com/74814780/191866394-e073f813-15bb-4b54-bb68-fe858786d811.png)